lightkurve/seismology/utils.py

```diff
@@ -26,7 +26,9 @@ def __repr__(self):
         return super().__repr__()
 
     def _repr_latex_(self):
-        return "{}: {} (method: {})".format(self.name, super()._repr_latex_(), self.method)
+        return "{}: {} (method: {})".format(self.name,
+                                    '$ {} $'.format(np.round(super().value,2)),
+                                    super().unit._repr_latex_(), self.method)
```

Comment:
This is @christinahedges I'm not sure if this is ideal, the actual value has more decimal places than the repr, so if I start doing math with this value, there will be additional decimal places that get used and might make my final value funny. Should we make the repr e.g. `3.45... ` instead of `3.45`? This problem is kind of small for now and will be taken care of when we can estimate errors.